src/components/GameView/SevenDoubleJacksDialog.vue

```diff
@@ -87,7 +87,11 @@ export default {
 			}
 		},
 		selectedJack() {
-			return this.selectedCardId === null ? null : (this.selectedCardId === this.topCard.id ? this.topCard : this.secondCard);
+			return this.selectedCardId === null
+				? null
+				: (this.selectedCardId === this.topCard.id
+					? this.topCard
+					: this.secondCard);
```

Comment:
Better to be formatted like this, but it kinda points out that the double ternary smells. What do you think? Legible?

Reply:
I guess this (below) is a touch better, since it makes the "early" exit condition a little more clear?  But then there's the age-old argument about early returns, but is this even that early?  I dunno, I don't have particularly strong feelings on the matter

```
		selectedJack() {
			if (this.selectedCardId === null) return null;
			return this.selectedCardId === this.topCard.id
				? this.topCard
				: this.secondCard;
		},
```

Reply:
I think the early return here is more clear; let's use it